NitroxClient/ClientAutoFacRegistrar.cs

```diff
@@ -14,6 +14,7 @@
 using NitroxClient.GameLogic.Bases.Spawning.BasePiece;
 using NitroxClient.GameLogic.ChatUI;
 using NitroxClient.GameLogic.FMOD;
+using NitroxClient.GameLogic.Helper;
```

Comment:
If we are removing a line of code do we need this dependency adding?

Reply:
removed